### ISSUE TITLE
Tensorflow 1.5.0?

### ISSUE THREAD

**Issue by bpasham** [greydanus/scribe#7]
hi, this is a great work. 

Is there any chances of making this work with Tensorflow 1.5.x?

Thanks

**Comment by greydanus** [comment]
Yeah, this could work with the newer TensorFlow but I don't have time to make the updates. If you (or someone else) does, please share!
